packages/tasks/src/model-libraries.ts

```diff
@@ -518,6 +518,20 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.whisperkit,
 		countDownloads: `path_filename:"model" AND path_extension:"mil" AND _exists_:"path_prefix"`,
 	},
+	hallo: {
+		prettyLabel: "Hallo",
+		repoName: "hallo",
+		repoUrl: "https://github.com/fudan-generative-vision/hallo",
+		docsUrl: "https://github.com/fudan-generative-vision/hallo?tab=readme-ov-file#hallo-hierarchical-audio-driven-visual-synthesis-for-portrait-image-animation",
+		countDownloads: `path_extension:"pth" OR path_extension:"onnx" OR path_extension:"ckpt" OR path_extension:"safetensors" OR path:"config.json"`,
```

Comment:
Is `config.json` always read/downloaded when instantiating a model with the library? If yes, it would be nice to only count `config.json` so that downloads are not duplicated. The problem with counting all files is that 1 user downloading the model will result in many counted downloads.

Reply:
Users may download models separately via file download button. So we want to statistic download counts by main model files.

Reply:
Unfortunately, that means that doing a git clone as recommended officially (https://github.com/fudan-generative-vision/hallo?tab=readme-ov-file#%EF%B8%8F%EF%B8%8F-usage) will lead to counting 8 downloads rather than 1. Is there any file/model that always need to be downloaded? E.g. hallo/net.pth?

Reply:
OK. I get it. I'll retain the main file only.